release/update-config.js

```diff
@@ -47,18 +51,18 @@ async function run() {
     .withConfigPath(path.resolve(__dirname, '..', 'helix-config.yaml'))
     .init();
 
-  const affected = [cfg.strains.get('default')];
+  const affected = [cfg.strains.get(strain)];
   let modified = false;
 
   // update package in affected strains
   console.log('Updating affected strains:');
   const packageProperty = `${WSK_NAMESPACE}/pages_${version}`;
-  affected.forEach((strain) => {
-    console.info(`- ${strain.name}`);
-    if (strain.package !== packageProperty) {
+  affected.forEach((s) => {
+    console.info(`- ${s.name}`);
+    if (s.package !== packageProperty) {
       modified = true;
       // eslint-disable-next-line no-param-reassign
-      strain.package = packageProperty;
+      s.package = packageProperty;
```

Comment:
@tripodsan we probably need to adjust this for universal runtime support, too

Reply:
yes. currently the universal support is disabled anyways, since the backend checks don't work... I'll look into it tomorrow